core/local/chokidar_watcher.js

```diff
@@ -98,8 +98,6 @@ module.exports = class LocalWatcher {
     this.watcher = chokidar.watch('.', {
       // Let paths in events be relative to this base path
       cwd: this.syncPath,
-      // Ignore our own .system-tmp-cozy-drive directory
-      ignored: /(^|[\/\\])\.system-tmp-cozy-drive/, // eslint-disable-line no-useless-escape
```

Comment:
We should keep using the chokidar option so we don't end up with `add` / `delete` events on download/copy.

Reply:
Loaded ignore rules [separately](https://github.com/cozy-labs/cozy-desktop/pull/1473/commits/4be40e130234eadb11d11829facb114ba4b8987a) in https://github.com/cozy-labs/cozy-desktop/pull/1473